docs/terms/data_docs.md

```diff
@@ -1,14 +1,100 @@
+
+### Versatility
+
+There are multiple use cases for displaying information in your Data Docs.  Three common ones are:
+
+1. Visualize all Great Expectations artifacts from the local repository of a project as HTML: Expectation Suites,
+   Validation Results and profiling results.
+1. Maintain a "shared source of truth" for a team working on a data project. Such documentation renders all the
+   artifacts committed in the source control system (Expectation Suites and profiling results) and a continuously
+   updating data quality report, built from a chronological list of validations by run id.
+1. Share a spec of a dataset with a client or a partner. This is similar to API documentation in software development.
+   This documentation would include profiling results of the dataset to give the reader a quick way to grasp what the
+   data looks like, and one or more Expectation Suites that encode what is expected from the data to be considered
+   valid.
+
+To support these (and possibly other) use cases Great Expectations has a concept of "data documentation site". Multiple sites can be configured inside a project, each suitable for a particular data documentation use case.
+
+### Customization
+
+The HTML documentation generated by Great Expectations Data Docs is fully customizable. If you would like to customize the look and feel of these pages or create your own, see our guide on [how to create Renderers for Custom Expectations](../guides/expectations/advanced/how_to_create_renderers_for_custom_expectations.md).
+
+## API basics
+
+### How to access
+
+Data Docs are rendered as HTML files.  As such, you can open them with any browser.
+
+### How to create
+
+If your Data Docs have not yet been rendered, you can create them from your Data Context.
+
+From the root folder of your Data Context, you can build your Data Docs with the CLI command:
+
+```bash title="Terminal command"
+great_expectations docs build
+```
+
+Alternatively, you can use your Data Context to build your Data Docs in python with the command:
+```python title="Python code"
+import great_expectations as ge
+context = ge.get_context()
+context.build_data_docs()
+```
+
+### Configuration
+
+Data Docs sites are configured under the `data_docs_sites` key in your deployment's `great_expectations.yml` file. Users can specify:
+
+- which <TechnicalTag relative="../" tag="datasource" text="Datasources" /> to document (by default, all)
+- whether to include Expectations, validations and profiling results sections
+- where the Expectations and validations should be read from (filesystem, S3, or GCS)
```

Comment:
Could we also Azure if we're including the other cloud providers? Same below.

Reply:
Added to both locations.